sdk/src/main/java/co/omise/android/models/APIError.kt

```diff
@@ -0,0 +1,18 @@
+package co.omise.android.models
+
+import org.joda.time.DateTime
+
+/**
+ * Represents Error object.
+ *
+ * @see <a href="https://www.omise.co/errors-api">Errors API</a>
+ */
+class APIError(
+        @JvmField
```

Comment:
If I understand correctly, `@JvmField` will not generate getter/setter. 

Why do you think we should add `@JvmField`?

Reply:
Same as the previous comment. One of the problems of going on holiday in the middle of a task is that you can't remember the logic behind certain things 🤦‍♂ Need to think on this one

Reply:
I thought ask asked me different thing.😂

From what I see now. If we call the field from Java it'll call `error.code` instead of Java convention `error.getCode()`.

Reply:
Yeah I get that, I just can't remember why I've made this decision, for all our data classes too 🤔